services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core/_events_user_services.py

```diff
@@ -98,7 +98,6 @@ async def create_user_services(app: FastAPI, scheduler_data: SchedulerData):
         node_id=scheduler_data.node_uuid,
         simcore_user_agent=scheduler_data.request_simcore_user_agent,
         swarm_stack_name=dynamic_services_scheduler_settings.SWARM_STACK_NAME,
-        run_id=scheduler_data.run_id,
     )
```

Comment:
waow. you are disasembling? ;)

Reply:
I am confused by this, why did this not appear in the original function?